ansible_ai_connect/main/settings/base.py

```diff
@@ -93,6 +94,11 @@
     "csp.middleware.CSPMiddleware",
 ]
 
+if os.environ.get("CSRF_TRUSTED_ORIGINS"):
```

Comment:
Just due to my ignorance... :) why do we need CSRF trusted origins? Why are they used for?

Reply:
This is what I do not understand 100% yet.  It is not needed when the AI Connect server is started from source, which uses Daphne for all endpoints. When the server is loaded from the image with podman-compose, it is needed.  I think it may be because we are using Daphne only for the /streaming_chat endpoint and others are processed with uwsgi, i.e. when the chatbot page is loaded, all HTML/Javascript/css, etc. are loaded through uwsgi, and the streaming_chat request goes to Daphne. I guess it breaks something on CSRF code used by Django, but I have not clearly figured it out yet.